app/subscriber/src/components/media-type-filters/MediaTypeFilters.tsx

```diff
@@ -2,22 +2,35 @@ import React, { useEffect, useState } from 'react';
 import { useContent, useLookup } from 'store/hooks';
 import { Button, ButtonHeight, ContentTypeName, IFilterSettingsModel } from 'tno-core';
 
-import { HomeFilterType } from '../constants';
+import { MediaFilterTypes } from './constants';
 import * as styled from './styled';
+import { determineStore } from './utils';
 
-export interface IHomeFilterProps {}
+export interface IMediaTypeFiltersProps {
+  filterStoreName:
```

Comment:
valid redux store names, could just use `string` and an error would be thrown when deconstructing anyways, but this provides better auto-completion

Reply:
tried `filterStoreName: keyof IContentState`; however, does not behave with dynamic deconstruction (line 31)